fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java

```diff
@@ -994,9 +994,10 @@ public static void loadJournal(Env env, JournalEntity journal) {
              * log a warning here to debug when happens. This could happen to other meta
              * like DB.
              */
-            LOG.warn("[INCONSISTENT META] replay failed {}: {}", journal, e.getMessage(), e);
+            LOG.warn("[INCONSISTENT META] replay failed {}: {}", journal, e.getMessage());
         } catch (Exception e) {
-            LOG.error("Operation Type {}", opCode, e);
+            LOG.error("Operation Type {}: {}", opCode, e.getMessage());
+            e.printStackTrace();
```

Comment:
Using `printStackTrace` would print the error msg to the stdout, i think it wouldn't help us debug. We can just log e so it will return the whole stack. If you are worried about the `Fast Throw` optimization that jvm would do, you can add `-XX:-OmitStackTraceInFastThrow` in start-fe.sh.

Reply:
> 

@ByteYue could you please submit a PR to add `-XX:-OmitStackTraceInFastThrow`? I think this is necessary

Reply:
> > 
> 
> @ByteYue could you please submit a PR to add `-XX:-OmitStackTraceInFastThrow`? I think this is necessary

OK, on my way.